src/Sql/dbo/Stored Procedures/OrganizationPasswordManager_Update.sql

```diff
@@ -0,0 +1,47 @@
+CREATE PROCEDURE [dbo].[OrganizationPasswordManager_Update]
+    @OrganizationId UNIQUEIDENTIFIER,
+    @Plan [nvarchar](50),
+    @PlanType [tinyint],
+    @Seats [int],
+    @MaxCollections [smallint],
+    @UseTotp [bit],
+    @UsersGetPremium [bit],
+    @Storage [bigint],
+    @MaxStorageGb [smallint],
+    @MaxAutoscaleSeats [int]
```

Comment:
I believe EDD requires update parameters to be nullable

https://contributing.bitwarden.com/server/mssql/edd/

Reply:
Update these to be nullable